docs/source/commandline/commandline-all.rst

```diff
@@ -47,57 +57,84 @@ Configuration commands
 
 
 
-Frontend commands
+Frontend Commands
 -----------------
 
 Note: By default, all the '**-list**' commands below write info to the screen. If you wish to write the info to a textfile instead, add this to the end of your command:
 
-  > filename
+  **> filename**
+
+...where 'filename' is the textfile's path and name (e.g., list.txt).
 
-...where 'filename' is the textfile's path and name (e.g., c:\\mame\\list.txt).
+Example:
+
+|  **mame -listcrc puckman > list.txt**
+|
+| This creates (or overwrites if it's already there) list.txt and fills the file with the results of **-listcrc puckman**. In other words, the list of each ROM used in Puckman and the CRC for that ROM are written into that file.
+|
 
+.. _mame-commandline-listxml:
 
 **-listxml** / **-lx** [*<gamename|wildcard>*]
 
 	List comprehensive details for all of the supported games. The output is quite long, so it is usually better to redirect this into a file. The output is in XML format. By default all games are listed; however, you can limit this list by specifying a driver name or wildcard after the -listxml command.
 
+.. _mame-commandline-listfull:
+
 **-listfull** / **-ll** [*<gamename|wildcard>*]
 
 	Displays a list of game driver names and descriptions. By default all games are listed; however, you can limit this list by specifying a driver name or wildcard after the **-listfull** command.
 
+.. _mame-commandline-listsource:
+
 **-listsource** / **-ls** [<*gamename|wildcard>*]
 
 	Displays a list of drivers and the names of the source files their game drivers live in. Useful for finding which driver a game runs on in order to fix bugs. By default all games are listed; however, you	can limit this list by specifying a driver name or wildcard after the **-listsource** command.
 
+.. _mame-commandline-listclones:
+
 **-listclones** / **-lc** [<*gamename|wildcard*>]
 
 	Displays a list of clones. By default all clones are listed; however, you can limit this list by specifying a driver name or wildcard after the **-listsource** command.
 
+.. _mame-commandline-listbrothers:
+
 **-listbrothers** / **-lb** [<*gamename|wildcard*>]
 
 	Displays a list of '*brothers*', or rather, other sets which are located in the same sourcefile as the gamename searched for.
 
+.. _mame-commandline-listcrc:
+
 **-listcrc** [<*gamename|wildcard*>]
 
 	Displays a full list of CRCs of all ROM images referenced by all drivers within MAME.
 
+.. _mame-commandline-listroms:
+
```

Comment:
This is another verb that can accept multiple patterns or system names.  It also accepts device names.

Reply:
Fixed on my side.